build_aux/ltmain.sh

```diff
@@ -1390,7 +1390,7 @@ scriptversion=2014-01-07.03; # UTC
 # GNU General Public License for more details.
 
 # You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
I saw many of these changes from `https` to `http`. Today, isn't it always better to use `http` rather than `https` ?

Reply:
The file without https are old ones - the PR has a lot of older files checked in.
In general needs to be split into its parts to be able to be reviewed and checked in accordingly, the first one would likely be to add handling of the `CONTROL` attribute - without the `WITH_EXTENDED_ACCDIS` parts.

The second one (big, libcob only) would be to add the key mapper.

Potentially the next would be the XAD-additions, but we'd know better when the previous changes would be merged in and working already.